miner/worker_builder.go

```diff
@@ -45,24 +45,22 @@ func (w *worker) fillTransactionsAndBundles(interruptCh chan int32, env *environ
 	{
 		bundles := w.eth.TxPool().PendingBundles(env.header.Number.Uint64(), env.header.Time)
 
-		// if no bundles, not necessary to fill transactions
-		if len(bundles) == 0 {
-			return errors.New("no bundles in bundle pool")
-		}
+		// if no bundles, skip bundle commit
+		if len(bundles) > 0 {
```

Comment:
What's the issue if we return error when bundle size is zero.

Reply:
yes, if return error here, that will be no txs included